extension/intents/tabs/tabs.js

```diff
@@ -556,3 +556,23 @@ if (!buildSettings.android) {
     },
   });
 }
+
+intentRunner.registerIntent({
+  name: "tabs.countTabs",
+  async run(context) {
+    context.keepPopup();
+    const tabs = await browser.tabs.query({ currentWindow: true });
+    const card = {
+      answer: {
+        num: `${tabs.length}`,
+        text: "Open tabs",
+        eduText: "Click mic and say 'gather all Google tabs'",
```

Comment:
It looks like we actually don't support "gather", though that could be added to the "collect" phrases that we have (`(collect | gather)`).

I noticed @awallin put single quotes around `'gather all Google tabs'` in the mockup, but generally we've been using double quotes (`"gather all Google tabs"`).

Also could you rename `num` to be `largeText`? It doesn't have to be a number, but it will be very large! I can imagine `mediumText` being added soon as well for some other information.

Reply:
I make updates to support "gather", use double quotes, and rename `num`.